src/PivotTableUI.jsx

```diff
@@ -367,19 +367,14 @@ class PivotTableUI extends React.PureComponent {
     );
   }
 
-  render() {
-    const numValsAllowed =
-      this.props.aggregators[this.props.aggregatorName]([])().numInputs || 0;
-
-    const rendererName =
-      this.props.rendererName in this.props.renderers
-        ? this.props.rendererName
-        : Object.keys(this.props.renderers)[0];
-
-    const rendererCell = (
+  rendererCell() {
```

Comment:
are the changes here just refactoring or are they required for the new renderer to function?